brut.j.dir/src/main/java/brut/directory/ZipUtils.java

```diff
@@ -30,6 +30,10 @@
 public class ZipUtils {
 
     private static Collection<String> mDoNotCompress;
+    
+    private ZipUtils(){
```

Comment:
`private ZipUtils() {`

Reply:
This is a Java best practice recommendation
http://www.javapractices.com/topic/TopicAction.do?Id=40

Reply:
Yes, but there is a space missing ;)

Reply:
I have add a commit with the space.